lib/src/DatePicker/components/Calendar.tsx

```diff
@@ -284,18 +294,42 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
           transKey={currentMonth!.toString()}
           className={classes.transitionContainer}
         >
-          <div>{this.renderWeeks()}</div>
+          <>
+            {(this.state.loadingQueue > 0 && (
+              <div className={classes.progressContainer}>
+                <CircularProgress />
```

Comment:
Must this render a CircularProgress? I believe this should be an option or at the very least able to be replaced by the user. @dmtrKovalenko What do you think?

Reply:
Good idea! I added support for an optional loadingIndicator property that would replace the CircularProgress